omnibus/Gemfile.lock

```diff
@@ -95,48 +93,6 @@ GEM
       train-core (~> 2.0, >= 2.0.12)
       tty-screen (~> 0.6)
       uuidtools (~> 2.1.5)
-    chef (15.2.20-universal-mingw32)
```

Comment:
?

Reply:
yeah that shouldn't be there at all, i think i did something in the wrong directory